composer.json

```diff
@@ -28,7 +28,7 @@
         "php" : "^7.0",
         "illuminate/config": "~5.1.24|~5.2.0|~5.3.0|~5.4.0",
         "illuminate/database": "~5.1.24|~5.2.6|~5.3.0|~5.4.0",
-        "illuminate/support": "~5.1.24|~5.2.0|~5.3.0|~5.4.0",
+        "illuminate/support": "~5.3.0|~5.4.0",
```

Comment:
Ok, drop support for older versions of `illuminate/config` and `illuminate/support` as well